src/lib.rs

```diff
@@ -11,7 +11,7 @@ the rust programming language.
 #![deny(unused_parens)]
 #![deny(non_upper_case_globals)]
 #![deny(unused_results)]
-#![warn(missing_docs)] // TODO: deny this
+#![deny(missing_docs)]
```

Comment:
Let’s keep this a `warn` and remove the `TODO`. I find it more convenient to just warn locally, and have it fail only on CI by denying warnings.